src/consensus/dkg.rs

```diff
@@ -114,6 +122,20 @@ impl DkgVoter {
             }
         }
 
+        // Special case: only one participant.
+        if elders_info.elders.len() == 1 {
```

Comment:
shall it also be checked that the `elders` is only self ?
or it has already be checked ?